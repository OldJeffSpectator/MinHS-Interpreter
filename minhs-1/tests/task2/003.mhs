main :: [Int]
       = let inc :: ([Int] -> [Int])
                   = letfun inc :: ([Int] -> [Int]) = (Cons 3);
         in inc (Cons 4 Nil);
