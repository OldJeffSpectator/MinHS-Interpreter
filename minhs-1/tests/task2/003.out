(Cons 3 (Cons 4 Nil))
